extras/travis_build.sh

```diff
@@ -56,6 +56,9 @@ if [ "$xPYTHON" = "1" ] ; then
    elif [ "$TRAVIS_PYTHON_VERSION" == "3.4.4" ]; then
       CONFIGURE_PYTHON="-DPYTHON_LIBRARY=/usr/lib/x86_64-linux-gnu/libpython3.4m.so -DPYTHON_INCLUDE_DIR=/opt/python/3.4.4/include/python3.4m/"
    fi
+   if [[ $OSTYPE == darwin* ]]; then
+      CONFIGURE_PYTHON="-DPYTHON-LIBRARY=/usr/lib/libpython2.7.dylib -DPYTHON_INCLUDE_DIR=/usr/local/Cellar/python@2/2.7.16/Frameworks/Python.framework/Versions/2.7/include/python2.7"
```

Comment:
```
CMake Warning:
  Manually-specified variables were not used by the project:
    PYTHON-LIBRARY
```

According to [FindPythonLibs](https://cmake.org/cmake/help/v3.0/module/FindPythonLibs.html) it should be:
```suggestion
      CONFIGURE_PYTHON="-DPYTHON_LIBRARY=/usr/lib/libpython2.7.dylib -DPYTHON_INCLUDE_DIR=/usr/local/Cellar/python@2/2.7.16/Frameworks/Python.framework/Versions/2.7/include/python2.7"
```

Reply:
Oh yes. Thanks. That was a typo :-(